Toggl.iOS/ViewControllers/Settings/SettingsViewController.cs

```diff
@@ -176,6 +174,23 @@ private IObservable<IImmutableList<SettingSection>> settingsSections()
 
             return sections.CombineLatest().Select(list =>
                 list.Where(CommonFunctions.NotNull).ToImmutableList());
+
+            SettingSection createWorkspaceSection(string workspaceName, Plan plan)
+            {
+                var workspaceRow = new NavigationRow(Resources.Workspace, workspaceName, ViewModel.PickDefaultWorkspace);
+
+                if (plan == Plan.Free)
+                    return new SettingSection("", new ISettingRow[]
+                    {
+                        workspaceRow,
+                        new NavigationRow(Resources.YourPlan, plan.Name(), ViewModel.OpenYourPlanSettings)
+                    });
+
+                return new SettingSection("", new ISettingRow[]
+                {
+                    workspaceRow
+                });
```

Comment:
This could be simplified a bit like:
```suggestion
                var rows = new List<ISettingRow> {
                    new NavigationRow(Resources.Workspace, workspaceName, ViewModel.PickDefaultWorkspace)
                }

                if (plan == Plan.Free)
                    rows.Add(new NavigationRow(Resources.YourPlan, plan.Name(), ViewModel.OpenYourPlanSettings));

                return new SettingSection("", rows.ToArray());
```